doc/Guide.md

```diff
@@ -100,4 +100,4 @@ This curve looks like an "S" with the top half bigger than the bottom. Mathemati
 ![MotivityExample](images/motivity_example.png)
 
 ## Further Help
-Further help and frequently asked questions can be found in the FAQ.
+Further help and frequently asked questions can be found in the ![FAQ](https://github.com/a1xd/rawaccel/blob/master/doc/FAQ.md).
```

Comment:
i think the exclamation mark in front of the link is used for images only, i will fix. thanks!